apps/kcapi-hasher.c

```diff
@@ -153,15 +153,17 @@ static void usage(char *name, int fipscheck)
 	if (fipscheck)
 		fprintf(stderr, "\t%s [-n BASENAME] [OPTION]... FILE\n", base);
 	else {
-		fprintf(stderr, "\t%s [-n BASENAME] [OPTION]... -c FILE\n", base);
+		fprintf(stderr, "\t%s [-n BASENAME] [OPTION]... -c FILE [-T FILE]\n", base);
 		fprintf(stderr, "\t%s [-n BASENAME] [OPTION]... FILE...\n", base);
 	}
 	fprintf(stderr, "\nOptions:\n");
 	fprintf(stderr, "\t-n --name\t\tForce given application name (sha512hmac/...)\n");
 	fprintf(stderr, "\t-S --self-sum\t\tPrint checksum of this binary and exit\n");
 	fprintf(stderr, "\t-L --self-sum-lib\tPrint checksum of the libkcapi library and exit\n");
-	if (!fipscheck)
+	if (!fipscheck) {
```

Comment:
why are these options left out in the fipscheck case?

Reply:
I believe its because  in programs like sha*hmac the hmac file is provided via -c and the program verifies the sums against files found in the hmac file. Fipscheck program doesnt expect there to be any filenames inside the hmac file. I guess thats why the option is disabled in fipscheck.